### ISSUE TITLE
Does not initially indicate if too much text selected

### ISSUE THREAD

**Issue by Gitoffthelawn** [rugk/offline-qr-code#240]
## Bug description
When creating a QR code from too much selected text, does not generate an error.

## Steps to reproduce
1.  Go to any page with plenty of text.
2.  Press Ctrl+A
3.  Generate a QR code

### Actual behavior
No error until you try to modify the text.

### Expected behavior
Error that string is too long.

## System
Operating system:  Windows
Browser and version: Firefox 72.0.2
Add-on version: 1.6

**Comment by rugk** [comment]
Cannot reproduce. If I use https://de.wikipedia.org/wiki/Wikipedia:Hauptseite and use the context menu to create the QR code, it does show me the error instantly.
Could you maybe specify the exact site, you've used?

![grafik](https://user-images.githubusercontent.com/11966684/73073041-c21ef480-3eb6-11ea-81ec-9603300ee122.png)

**Comment by Gitoffthelawn** [comment]
That's funny, I tested it on Wikipedia too! :)

Here's the page I used: https://en.wikipedia.org/wiki/QR_code

I only got the error after I tried to modify the text.

I don't recall if I had the toolbar button on the in the overflow menu or not during that test.  I have to go right now, otherwise I would check it for you.

**Comment by rugk** [comment]
Okay, now can reproduce the issue with both Wikipedia pages.
Strangely, I could not with the dev version before. That is strange…

Maybe a timing issue/race condition?

On Fedora 31, Firefox 72.

**Comment by Gitoffthelawn** [comment]
Yes, it could be a race condition.  It happens every time for me, which is interesting that it doesn't do the same for you.  I'm glad you mentioned that it happens on both pages for you.

**Comment by rugk** [comment]
Yeah repdocuible on the device I mentioned above, but not on another Windows device. 🤔

**Comment by Gitoffthelawn** [comment]
Same version of Windows and browser on each?

**Comment by rugk** [comment]
Actually no, only reproducible on Linux ([Fedora](https://github.com/rugk/offline-qr-code/issues/240#issuecomment-578514275)), not on Windows here. Same Firefox though. 🤔

**Comment by muxator** [comment]
Hi,

confirmed on Ubuntu 20.04, Firefox 77.0.1, Offline QR Code Generator 1.7 (checked via about:telemetry#environment-data-tab_addons as I do not know any other way to see the installed version).

**Comment by Gitoffthelawn** [comment]
@muxator Thank you for the confirmation.  Mozilla has made it a little less obvious where to quickly find the version numbers for installed extensions.  The trick is to go to `about:addons`, and then hover the cursor over the name of the extension.  The extension's version should pop up in a tooltip.

**Comment by rugk** [comment]
…or open the extension page details and scroll down to the bottom. There, eveything is.

**Comment by Gitoffthelawn** [comment]
@rugk That too! :)

**Comment by muxator** [comment]
I checked the extension version using the three methods (yay). It's **1.7**, indeed.

Here's a screen capture when replicating the issue. The context menu got cut off. The command I'm invoking is "`QR code from selection`".


![Peek 2020-06-30 23-27](https://user-images.githubusercontent.com/4067621/86178937-98cf5800-bb29-11ea-9258-84941a5e9568.gif)


- kubuntu 20.04
- firefox 77.0.1
- offline qr code 1.7
